# Publish all joint states
joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

# Control all joints
arm_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - link0_link1x_joint
    - link1x_link1y_joint
    - link1y_link2x_joint
    - link2x_link2y_joint
    - link2y_link3x_joint
    - link3x_link3y_joint
    - link3y_link4x_joint
    - link4x_link4y_joint
    - link4y_link5x_joint
    - link5x_link5y_joint

  gains:
    link0_link1x_joint: {p: 100,  d: 1, i: 0.1}
    link1x_link1y_joint: {p: 100,  d: 1, i: 0.1}
    link1y_link2x_joint: {p: 100,  d: 1, i: 0.1}
    link2x_link2y_joint: {p: 100,  d: 1, i: 0.1}
    link2y_link3x_joint: {p: 100,  d: 1, i: 0.1}
    link3x_link3y_joint: {p: 100,  d: 1, i: 0.1}
    link3y_link4x_joint: {p: 100,  d: 1, i: 0.1}
    link4x_link4y_joint: {p: 100,  d: 1, i: 0.1}
    link4y_link5x_joint: {p: 100,  d: 1, i: 0.1}
    link5x_link5y_joint: {p: 100,  d: 1, i: 0.1}